custom_components/tuya_local/devices/windcalm_fan.yaml

```diff
@@ -0,0 +1,33 @@
+name: Windcalm Ceiling Fan
+primary_entity:
+  entity: fan
+  dps:
+    - id: 60
+      type: boolean
+      name: switch
+    - id: 62
+      type: integer
```

Comment:
Is it really an integer, or is it a string?
If an integer, it might be better to map using a range and scale instead of listing all the options.  The scale is a divisor, so since the HA target is 0 - 100, the scale for 6 speeds should be 0.06.

Reply:
I'll look into it.

Reply:
A bit late, but I was enjoying a little holiday.

I updated the speed mappings. Works like a charm.